### PR TITLE
Add method for activity summary

### PR DESCRIPTION
**Pull request by bradfordboyle**

### CODE REVIEW

**Comment by bradfordboyle** [comment]
@anxiousmodernman For some reason I cannot request a review from you on this PR. Would you mind giving this a once over?

File: src/errors.rs [hunk 1]
```diff
@@ -23,23 +27,29 @@ pub enum Error {
...
+
+impl From<chrono::ParseError> for Error {
+    fn from(kind: chrono::ParseError) -> Self {
+        Error::DateParse(kind)
+    }
 }
 
 impl From<reqwest::Error> for Error {
-    fn from(kind: reqwest::Error) -> Error {
+    fn from(kind: reqwest::Error) -> Self {
```
Comment (anxiousmodernman) [inline src/errors.rs:40]: :+1:

File: src/date.rs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+use std::fmt;
+use std::str::FromStr;
+
+use errors;
+
+#[derive(Debug)]
+pub struct Date(::chrono::NaiveDate);
```
Comment (anxiousmodernman) [inline src/date.rs:7]: chrono has the timezone-aware `Date<Tz>`, but we can use aliases to get around it if we end up needing it.

**Review by anxiousmodernman — APPROVED** [review]
LGTM

**Pull request merged**